Doc/library/ipaddress.rst

```diff
@@ -117,6 +116,17 @@ write code that handles both IP versions correctly.  Address objects are
    >>> ipaddress.IPv4Address(b'\xC0\xA8\x00\x01')
    IPv4Address('192.168.0.1')
 
+   .. versionchanged:: 3.8
+
+      Leading zeros are tolerated, even in ambiguous cases that look like
+      octal notation.
+
+   .. versionchanged:: 3.8.9/3.9.3
```

Comment:
I tried multiple approaches. This seems to be the best way to encode multiple versions without messing up the output. Neither comma-separated values nor multiple versionchanged entries result in proper output.

Reply:
In master, put 3.10. In 3.9, put 3.9.x. etc.

See what was done for similar fixes, like https://bugs.python.org/issue42967